Tests/Symbols.cs

```diff
@@ -139,9 +144,9 @@ private static Symbol CreateCfdSymbol(string symbol, string market)
             return Symbol.Create(symbol, SecurityType.Cfd, market);
         }
 
-        private static Symbol CreateOptionSymbol(string symbol, OptionRight right, decimal strike, DateTime expiry)
+        private static Symbol CreateOptionSymbol(string symbol, OptionRight right, decimal strike, DateTime expiry, string market = Market.USA)
         {
-            return Symbol.CreateOption(symbol, Market.USA, OptionStyle.American, right, strike, expiry);
+            return Symbol.CreateOption(symbol, market, OptionStyle.American, right, strike, expiry);
         }
```

Comment:
Added `Tests/TestData/FuturesExpiryFunctionsTestData.xml` bellow is an empty file?